types/puppeteer/index.d.ts

```diff
@@ -2249,3 +2308,50 @@ export function executablePath(): string;
 export function launch(options?: LaunchOptions): Promise<Browser>;
 /** This methods attaches Puppeteer to an existing Chromium instance. */
 export function createBrowserFetcher(options?: FetcherOptions): BrowserFetcher;
+/**
+ * Predefigned puppeter devices
+ * 
+ * see DeviceDescriptors.js for full list
+ */
+export const devices: {
+  [key: string]: Device;
+};
+export const error: Error | undefined;
+
+
+
+export type KeysF = 'F1' |  'F2' | 'F3' | 'F4' | 'F5' | 'F6' | 'F7' | 'F8' | 'F9' |'F10' | 'F11' | 'F12' | 'F13' | 'F14' | 'F15' | 'F16' | 'F17' | 'F18' | 'F19' |'F20' | 'F21' | 'F22' | 'F23' | 'F24';
```

Comment:
Do we really want to hardcode all of these constants, what about non-US layouts?

Reply:
Autocompletion work fine :)

Reply:
I only hardcode the Keys hardcoded in `USKeyboardLayout.js`,
About the non-US layout, since it is not available inside puppeteer code, with or without this typing that will not change anything.
If you wan to use a non US key, you will trigger a assert from `input.js`: 
    assert(definition, `Unknown key: "${keyString}"`);
the current puppeteer public interface do not allow non US typing.